bamboo/docker-compose-local.yml

```diff
@@ -9,20 +9,20 @@ services:
       - ../packages/test-data/keys/ssh_client_rsa_key.pub:/ssh_client_rsa_key.pub
       - ../packages/test-data:/data
   build_env:
-    image: cumuluss/cumulus-build-env:latest
+    image: cumuluss/cumulus-build-env:test
     volumes:
       - ../:/source/cumulus
       - ../packages/test-data:/tmp/cumulus_unit_test_data
     ports:
-      - "20:20"
-      - "21:21"
-      - 47400-47470:47400-47470
-      - 3030:3030
-      - 2222:2222
-      - 4550-4599:4550-4599
-      - 5432:5432
-      - 8080:8080
-      - 9200:9200
+      - "127.0.0.1:20:20"
```

Comment:
This is a security update to disallow port access from hosts other than localhost.